### PR TITLE
Adding StaticLib 'How To' doc

### PR DESCRIPTION
**Pull request by jasonstephen15**

### CODE REVIEW

**Comment by liminzhu** [comment]
Some general feedback,

* Let's clearly spell out WebView2Loader.dll when you're referring to the load dll, so there is no ambiguity what file we're referring to.
*  `Generally, the Loader DLL is a file that interacts with an app to find the WebView runtime, or edge binary on the box. ` to `WebView2Loader.dll is a small component that helps apps locate the WebView2 Runtime (or non-stable Microsoft Edge channels) on the device.`
* `Microsoft Edge WebView team`. WebView**2** team.
* Add ToC 

(I'm also assuming you've talked to Diana about the Procedure part, since I actually haven't looked into how this works :))

File: microsoft-edge/webview2/howto/static.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- The WebView2 SDK contains a header file, loader.dll., and the IDL file. Generally, the Loader DLL is a file that interacts with an app to find the WebView runtime, or edge binary on the box. 
+
+For apps that have a single executable, and do not want to ship a loader.dll, follow the process below.
+
+## Procedure
+
+1. Open up your app's `.vcxproj` project file in a text editor, like VSCode.
+    > [!NOTE]
+    > The .vcproj project file may be a hidden file, meaning you wouldn't see it in Visual Studio. Use a terminal to find this file.
+1. Locate the section in the code where you would include the WebView2 NuGet package target files. This section can be found after the < /target > tag.
```
[the remark this answers is not in the document]
Comment (jm-trd-ms) [reply microsoft-edge/webview2/howto/static.md]: Updated.

```md
1.  Copy the following code snippet and paste it everywhere the `Microsoft.Web.WebView2.targets` is included.  

    > [!NOTE]
    > For example, include it after the following code block.  
    > 
    > ```csharp
    > <Import Project="..\packages\Microsoft.Web.WebView2.0.9.579-prerelease\build\native\Microsoft.Web.WebView2.targets" Condition="Exists('..\packages\Microsoft.Web.WebView2.0.9.579-prerelease\build\native\Microsoft.Web.WebView2.targets')" />
    > ```  
    
    ```csharp
    <PropertyGroup> <WebView2LoaderPreference>Static</WebView2LoaderPreference> </PropertyGroup>
    ```
```

File: microsoft-edge/webview2/toc.yml [hunk 2]
```diff
@@ -36,6 +36,8 @@
   href: roadmap.md
 - name: How to
   items:
+    - name: Link Static Loader
```
Comment (jm-trd-ms) [inline microsoft-edge/webview2/toc.yml:39]: Is this a "Link static loader" or a "Static link loader" or something else?

**Review by jm-trd-ms — CHANGES_REQUESTED** [review]
I updated and reviewed the main content and left a comment about the toc listing.

File: microsoft-edge/webview2/toc.yml (hunk 2 above)
Comment (jasonstephen15) [reply microsoft-edge/webview2/toc.yml:39]: It is (How to) link the static loader, I just removed "the"

Comment (jasonstephen15) [reply microsoft-edge/webview2/toc.yml:39]: Actually, the loader is what we are trying to link, in a static manner (with the package), does this make sense? I think "Statically link loader" works?

Comment (jm-trd-ms) [reply microsoft-edge/webview2/toc.yml]: Updated.

**Review by jm-trd-ms — APPROVED** [review]

**Comment by jm-trd-ms** [comment]
#sign-off offline and merged.

**Pull request merged**